SPECS/tdnf/tdnf.spec

```diff
@@ -28,17 +33,22 @@ BuildRequires:  glibc-devel
 BuildRequires:  gpgme-devel
 BuildRequires:  libmetalink-devel
```

Comment:
IIRC `libmetalink` is not required for the build anymore.

https://github.com/vmware/tdnf/pull/266

Reply:
Seems to be the case in the currently existing metalink plugin code, which has no dependency on libmetalink:

https://github.com/vmware/tdnf/blob/v3.5.2/plugins/metalink/CMakeLists.txt

We can go back later and remove libmetalink from the toolchain (since it's unmaintained, with a last release in 2015), but it'd be nice to get it out of the spec.